### DISCUSSION TITLE
Enemy Levelling

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#228] [Ideas]
Instead of levelling up enemies uniformly, distribute enemy levels from a pool to enemies randomly. This randomness can be useful information to add to predicts.

For enemy spawning: determine a "expected enemy party size" then roll a "level pool size divided by expected enemy party size" as the chance to increase the enemy level on spawn.

**Comment by doubledh** [comment]
1. does that.... merge in the hp randomization?
2. Does that make enemy atk's rather random?
3. may it cause a problem if it sucks all the power from one and puts it into the other, making the smol one just disintegrate on its own? (or does that just make scenarios more diverse?)

**Reply by ntseng** [reply]
1. I think we should still have HP Randomization on top of level randomization so that players with HP read can tie-break "which of these level 2 enemies should we focus first?"
2. Hopefully most enemy level ups can come with new skills instead of only stats, but yes danger/enemy is supposed to vary based on level randomization.
3. If we find the bonus levels are being assigned too clumpily, I'd first want to investigate changing the distribution algorithm from pure random assignment.
